changelog.d/1905.change.rst

```diff
@@ -0,0 +1 @@
+Fixed test failure introduced in 41.6.0 when the 'tests' directory is not present.
```

Comment:
test fail? original code in _imp.py was broken.

Reply:
But you're right, that message is a little unclear, implicating the test and not the code. I can improve that.